lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -89,6 +93,13 @@ UsdSceneItem::Ptr UsdUndoRenameCommand::renamedItem() const { return _ufeDstItem
 
 bool UsdUndoRenameCommand::renameRedo()
 {
+    // If the new name is the same as the current name, do nothing.
+    // This is the same behavior as the Maya rename command for Maya nodes.
+    if (_unchanged) {
+        _ufeDstItem = _ufeSrcItem;
```

Comment:
Why can't we make this assignment in the constructor?  It never changes, right?  Also, why would we only be doing it in redo()?

Reply:
Ah, yes, it could be done in the constructor. I was mimicking the behavior of the other cases that only set it in  redo and 
undo.